File: CS/CustomGridControl/MyGridView.cs
using DevExpress.Data;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace T263441 {
    public class MyGridView : DevExpress.XtraGrid.Views.Grid.GridView {
        public MyGridView() : this(null) { }
        public MyGridView(DevExpress.XtraGrid.GridControl grid)
            : base(grid) {

        }
        protected override string ViewName { get { return "MyGridView"; } }
        protected override BaseGridController CreateDataController() { return new MyDataController(); }
    }
}

File: VB/CustomGridControl/MyGridView.vb
Imports DevExpress.Data
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace T263441
    Public Class MyGridView
        Inherits DevExpress.XtraGrid.Views.Grid.GridView

        Public Sub New()
            Me.New(Nothing)
        End Sub
        Public Sub New(ByVal grid As DevExpress.XtraGrid.GridControl)
            MyBase.New(grid)

        End Sub
        Protected Overrides ReadOnly Property ViewName() As String
            Get
                Return "MyGridView"
            End Get
        End Property
        Protected Overrides Function CreateDataController() As BaseGridController
            Return New MyDataController()
        End Function
    End Class
End Namespace
